terraform/webhook-template.tf

```diff
@@ -0,0 +1,27 @@
+resource "azurerm_template_deployment" "github_management_template" {
```

Comment:
shouldn't you use a this:
https://registry.terraform.io/providers/hashicorp/azurerm/latest/docs/resources/resource_group_template_deployment

this one is deprecated

Reply:
ammended to use this instead